pkg/reconciler/openshift/tektonaddon/testdata/test-versioned-resolver-name-expected.yaml

```diff
@@ -0,0 +1,46 @@
+# auto generated by hack/openshift/fetch-tektoncd-catalog-tasks.sh
+# DO NOT EDIT: use the script instead
+# source: https://raw.githubusercontent.com/openshift/tektoncd-catalog/release-v0.10/buildah/buildah.yaml
+#
```

Comment:
I guess this test data is not managed by `hack/openshift/fetch-tektoncd-catalog-tasks.sh`?

Reply:
Yes 
i copied from another example where it was mentioned
but yeah removed those comments